README.md

```diff
@@ -53,6 +49,8 @@ Make sure to install the Hugo extended version specified by the `HUGO_VERSION` e
 To build and test the site locally, run:
 
 ```bash
+# install dependencies
+npm ci
```

Comment:
If someone doesn't do this, what doesn't work?

Reply:
the build fails.